### PR TITLE
levelup docs say to use gte in instead of start

### PR DESCRIPTION
**Pull request by Trott**
https://github.com/rvagg/node-levelup#createReadStream says to use `gte` instead of `start` so updated the problem/solution to do that.

### CODE REVIEW

**Comment by rvagg** [comment]
I can't recall off the top of my head if the version of level we are including in levelmeup even has support for the newer properties, did you try it out with the version shipped with levelmeup?

**Comment by ralphtheninja** [comment]
It uses `levelup@0.18.6`

```
level@0.18.0 node_modules/level
├── level-packager@0.18.0 (levelup@0.18.6)
└── leveldown@0.10.2 (bindings@1.1.1, nan@0.6.0)
```

[`createReadStream()`](https://github.com/rvagg/node-levelup/tree/213e989e2b75273e2b44c23f84f95e35bff7ea11#dbcreatereadstreamoptions) docs for this version.

**Comment by ralphtheninja** [comment]
I guess this PR is in order, but `level` would have to be updated first imo.

**Comment by Trott** [comment]
@rvagg I tried the solutions with the version of `level` that was installed when I installed `levelmeup` and it seems to work. But the version is as @ralphtheninja and the documentation there is as he says. So I'm not sure the problem is between keyboard and chair or if there's something going on that allows `gte` and `lte` to work in that version even though it's not documented.

**Comment by martinheidegger** [comment]
Level has been updated in #79, this PR needs a rebase.

**Comment by Trott** [comment]
OK, rebased and force pushed.

File: exercises/horse_js_tweets/problem.en.md [hunk 1]
```diff
@@ -50,23 +50,23 @@ then your query will end at the entry with a key that would come
 before your end key in sorted order.
 
 To target a range, you supply an options object to
-`createReadStream()` with `start` and/or `end` properties that
-define your range keys:
+`createReadStream()` with `gte` (greter than or equal) and/or `lte` (less than
```
Comment (martinheidegger) [inline exercises/horse_js_tweets/problem.en.md]: "greater", typo

**Comment by Trott** [comment]
Typo fixed.

**Pull request merged**